modules/es/ut-coverage/es_UT.c

```diff
@@ -39,6 +39,7 @@
 #include "es_UT.h"
 #include "target_config.h"
 #include "cfe_config.h"
+#include "cfe_es_api_typedefs.h"
```

Comment:
Also not needed, as "es_UT.h" pulls in everything for ES, including the API typedefs.

Reply:
Good to know - I saw different approaches (including and not including headers recursively).
I think it's better to avoid when not necessary as well.
Will remove where not required.